.github/workflows/health_metrics.yml

```diff
@@ -313,31 +313,6 @@ jobs:
             cd scripts/health_metrics/generate_code_coverage_report
             swift run CoverageReportGenerator --presubmit "firebase/firebase-ios-sdk" --head-commit "${GITHUB_SHA}" --token $(gcloud auth print-identity-token) --xcresult-dir "/Users/runner/test/codecoverage" --log-link "https://github.com/firebase/firebase-ios-sdk/actions/runs/${GITHUB_RUN_ID}" --pull-request-num ${{github.event.pull_request.number}} --base-commit "$base_commit"
           fi
-      - name: Incremental Code Coverage
```

Comment:
I'm ok with this.

Could also be ok to comment out with a GitHub Issue number to add it back.

Reply:
SGTM